packages/block-library/src/quote/edit.js

```diff
@@ -48,6 +48,7 @@ export default function QuoteEdit( { attributes, setAttributes, isSelected, merg
 								citation: nextCitation,
 							} )
 						}
+						noFocusOnMount={ true }
```

Comment:
It looks like `noFocusOnMount` prop isn't part of the public API as of today. @ellatrix, can you think of another way of handling that which would be useful in web context as well? It would be great to avoid having diverging APIs on the web and mobile.

Aside:
```suggestion
						noFocusOnMount
```
should work the same.

Reply:
👋 @gziolo , thanks for chipping in here!

I had noticed that there is a `focusOnMount` prop in other places but, I wasn't sure if it's appropriate for the case here so, I went with introducing the new one. Happy to revise if it won't be diluting the meaning of `focusOnMount`.

Reply:
I followed your suggestion to simplify passing the prop 👍 . Done with 863a20d.

Reply:
> It looks like noFocusOnMount prop isn't part of the public API as of today.

@gziolo , not sure, is that a blocker for the PR? Just trying to understand if there's more work to be done on this PR in order to merge it.

Reply:
Conversation resumed here: https://github.com/WordPress/gutenberg/pull/15878#discussion_r288918974.